src/gluonts/shell/__main__.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import click
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.command()
+def serve():
+    from gluonts.shell.serve import DefaultShell
+    DefaultShell().run()
+
+
+FORECASTER_BY_NAME = {
+    'deepar': 'gluonts.model.deepar.DeepAREstimator',
+    'r': 'gluonts.model.r_forecast.RForecastPredictor',
+}
+
+
+@cli.command()
+@click.option(
+    "--data-path",
+    type=click.Path(exists=True),
+    envvar="SAGEMAKER_DATA_PATH",
```

Comment:
If the goal is to make the shell SageMaker-agnostic, maybe the envvar should be called `GLUONTS_DATA_PATH`.

Reply:
This is the variable name used by official 1P algorithms, so I wanted to be compliant here.

Reply:
I understand.